### PR TITLE
bootstrap added

### PR DESCRIPTION
**Pull request by qik-in-mate**
bootstrap added

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    <div class="container">
+      <h1 class="my-2">Personal Information</h1>
+      <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post" id="form">
+        <div class="my-3">
+          <fieldset>
+            <legend>Personal Information</legend>
+            <div class="row">
+              <div class="form-group col-12 col-sm-6 col-lg-8">
+                <label for="usersurname">Surname:</label>
+                <input class="form-control" id="usersurname" type="text" name="surname">
```
Comment (carrot-e) [inline index.html]: Id `user-surname` would be better.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+          </fieldset>
+        </div>
+        <div class="my-2 row">
+          <fieldset class="form-group col-lg-6 offset-lg-2 col-12 col-sm-6 offset-sm-3">
+            <legend>Your comments:</legend>
+            <textarea class="form-control" id="comments" name="comments" rows="3" placeholder="Comments..." minlength="10" maxlength="200"></textarea>
+          </fieldset>
+        </div>
+        <div class="row text-right">
+          <div class="col-8 col-sm-8 col-md-8 col-lg-8"></div>
```
Comment (carrot-e) [inline index.html]: Why not just `col-8`?

**Review by carrot-e — COMMENTED** [review]
screenshot from 2018-12-20 17-57-34
1. XS screen requirement is not implemented.
2. Could you add some space below the "Submit" button?

File: index.html (hunk 2 above)
Comment (qik-in-mate) [reply index.html]: Right You are! Edited.

File: index.html (hunk 1 above)
Comment (qik-in-mate) [reply index.html]: All Id's values edited.

File: index.html [hunk 3]
```diff
@@ -114,10 +114,10 @@ <h1 class="my-2">Personal Information</h1>
...
           </fieldset>
         </div>
-        <div class="row text-right">
-          <div class="col-8 col-sm-8 col-md-8 col-lg-8"></div>
-          <div class="col-4 col-sm-4 col-md-4 col-lg-4">
-            <input class="btn btn-success btn-block" type="submit" value="Submit">
+        <div class="my-2 row text-right">
+          <div class="col-sm-8 "></div>
+          <div class="col-12 col-sm-4">
+            <input class="btn btn-success btn-block " type="submit" value="Submit">
```
Comment (carrot-e) [inline index.html]: You don't need a space before the closing `"`. (Same at line 118)

Comment (qik-in-mate) [reply index.html]: Well noted! Updated!

**Review by carrot-e — APPROVED** [review]
LGTM :+1:

**Pull request merged**